app/src/main/java/net/bonysoft/magicmirror/MainActivity.java

```diff
@@ -137,4 +155,40 @@ private DecimalFormat getTemperatureFormat() {
         temperatureFormat.setRoundingMode(RoundingMode.HALF_DOWN);
         return temperatureFormat;
     }
+
+    private final TwitterModule.TwitterListener tweetListener = new TwitterModule.TwitterListener() {
+
+        @Override
+        public void onNextTweet(final Status tweet) {
+            runOnUiThread(new Runnable() {
+                @Override
+                public void run() {
+                    displayTweet(tweet);
+                }
+            });
+        }
+    };
+
+    private void displayTweet(Status tweet) {
+        SpannableStringBuilder userHandle = highlightUserHandle(tweet.getUser().getScreenName());
+
+        final CharSequence text = TextUtils.concat(
+                userHandle,
+                "\n",
+                tweet.getText()
+        );
+
+        final AlphaAnimation fadeIn = new AlphaAnimation(0.0f, 1.0f);
+        fadeIn.setDuration(getResources().getInteger(android.R.integer.config_shortAnimTime));
+        tweetLabel.startAnimation(fadeIn);
+        tweetLabel.setText(text);
+    }
+
+    @NonNull
+    private SpannableStringBuilder highlightUserHandle(String handle) {
+        SpannableStringBuilder userHandle = new SpannableStringBuilder("@" + handle);
+        StyleSpan iss = new StyleSpan(Typeface.BOLD_ITALIC);
```

Comment:
Will rename the variable.
It's bold AND italic because of my crappy designer taste 👅 
(it looked nice....)